libpod/runtime.go

```diff
@@ -176,14 +193,15 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 	runtime.netPlugin = netPlugin
 
 	// Set up the state
-	if runtime.config.InMemoryState {
+	switch runtime.config.StateType {
+	case InMemoryStateStore:
 		state, err := NewInMemoryState()
 		if err != nil {
 			return nil, err
 		}
 		runtime.state = state
-	} else {
-		dbPath := filepath.Join(runtime.config.StaticDir, "state.sql")
+	case SQLiteStateStore:
+		dbPath := filepath.Join(runtime.config.StaticDir, "sql_state.db")
```

Comment:
This seems to break all existing SQL state records. Is that intentional?

Reply:
Yes, it is. This is going to be an breaking DB change anyways as we swap over to BoltDB by default, so I renamed the state something a little more clear (.sql usually indicates SQL scripts and not databases)